stat/sampleuv/example_rate_test.go

```diff
@@ -24,26 +24,17 @@ func ExampleMetropolisHastings_samplingRate() {
 	// Successive samples are correlated with one another through the
 	// Markov Chain defined by the proposal distribution. To get less
```

Comment:
"To get *fewer*"

Reply:
No, less is the correct word here. You get the same number of samples (`len(samples)`), but the samples are not as correlated, since more steps in the Markov chain are taken. 

Fixing the language to help clarify.